sql/src/main/java/io/crate/execution/engine/collect/files/S3FileInput.java

```diff
@@ -95,7 +95,7 @@ public InputStream getStream(URI uri) throws IOException {
         if (object != null) {
             return object.getObjectContent();
         }
-        return null;
+        throw new IOException("Failed to load S3 URI: " + uri.toString());
```

Comment:
Would be nice if we could include more information in the error message.

Reply:
any suggestion how?